src/System.Runtime.Extensions/tests/System/IO/Path.cs

```diff
@@ -61,22 +95,50 @@ public static void GetExtension(string expected, string path)
     [Fact]
     public static void GetFileName()
     {
+        Assert.Equal(null, Path.GetFileName(null));
```

Comment:
If [InlineData] can handle params you could send it all through Path.Combine(params string[] paths); though that's a really inefficient-looking method when paths.Length == 1.

And there's always good old [MemberData].

Certainly not a critical issue, more of the food-for-thought category.